.github/workflows/tests.yml

```diff
@@ -117,8 +125,10 @@ jobs:
           node-version: 18
           cache: "yarn"
           cache-dependency-path: yarn.lock
+      - name: Enable Corepack
+        run: corepack enable
```

Comment:
Out of curiosity, why do we need this?

Reply:
It is the recommended way to install Yarn v2+ according to the docs: https://yarnpkg.com/getting-started/install

Reply:
Makes sense, thanks a lot for the explanation 🙏